config/initializers/paperclip.rb

```diff
@@ -15,7 +15,7 @@
         access_key_id: ENV['S3_ACCESS_KEY_ID'],
         secret_access_key: ENV['S3_SECRET_ACCESS_KEY']
       },
-      path: path,
+      path: "#{ENV['S3_PATH']}/#{path}",
```

Comment:
Is this a breaking change? What will happen to existing supermarket's that upgrade and don't specify 'S3_PATH'? Won't that add a preceding '/' to the path?